projects/plugins/protect/src/js/routes/firewall/index.jsx

```diff
@@ -520,7 +498,6 @@ const FirewallPage = () => {
 					{ ( jetpackWafIpAllowListEnabled || ipAllowListHasContent ) && (
 						<div className={ styles[ 'manual-rules-section' ] }>
 							<Textarea
-								id="jetpack_waf_ip_allow_list"
```

Comment:
Same as in the above, `id` is required to make the field functional.

Reply:
Restored!